src/response.rs

```diff
@@ -129,10 +149,7 @@ enum Decoder {
     },
     /// An error occured reading the Gzip header, so return that error
     /// when the user tries to read on the `Response`.
-    Errored {
-        err: Option<io::Error>,
-        head: Head,
-    }
+    Errored { err: Option<io::Error>, head: Head },
```

Comment:
So, it made all structs in patterns on different lines, but made this struct variant a single line. Would rather keep it on multiple lines.